hw_05.09.23/task2/exc_try.py

```diff
@@ -0,0 +1,19 @@
+def input_errors():
+    try:
+        first = int(input("Enter first number: "))
+        second = int(input("Enter second number: "))
+
+        result = first ** 2 / second
+
+        print(result)
+        return result
+
+    except ZeroDivisionError:
+        print("You can't divide by 0")
+    except ValueError:
+        print("Enter the number!")
+    except IndexError:
```

Comment:
There's no way the block of code in `try` could have raised and IndexError (we don't work with indices in this block)